dashboard/server/README.md

```diff
@@ -0,0 +1,22 @@
+# StratoCyberLab Dashboard - Server
+
+## Development
+
+- `pip install -r requirements.txt` to install dependencies
+- `python app.py` to run the development server (also done in Docker)
+
+## Code Quality
+
+Before committing your changes, please make sure they do not generate new 
+warnings (contain lints) and the code is properly formatted.
+
+### Linting
+
+We use [pylint](https://pypi.org/project/pylint) to check server code for lints.
+Use the command `pylint .` (in this directory) to run code analysis.
+
+### Formatting
+
+[autopep8](https://pypi.org/project/autopep8) is used for code formatting.
+Use the command `autopep8 --in-place *.py` (in this directory) to reformat 
```

Comment:
I propose to add `--max-line-length N` where N is atleast 120. The pep8 standard for max 80 characters on a line is a bit outdated nowadays with ultra resolution monitors imo 😃

Reply:
Okay, I migrated the pylint config to `pyproject.toml` and included the max-line-length there